masterlist.yaml

```
@@ -2113,6 +2113,35 @@ plugins:
             text: 'Приставката е стара - изисква се само ако менюто за конфигуриране на модификациите не работи.'
         condition: 'file("Mart''s Mutant Mod - Master Menu Module.esp") or checksum("Mart''s Mutant Mod.esm", DC2E29E4)'
 
+  # Raiders Redone and Modules #
+  - name: '(RR(Females|GoreGrafitti)Removed|RaidersRedoneModular01)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout3/mods/2466/'
+        name: 'Raiders Redone'
+  - name: 'RaidersRedoneModular01.esp'
+    tag:
+      - Actors.ACBS
+      - Delev
+      - Factions
+      - Invent
+      - Names
+    clean:
+      - crc: 0xA58559F0
+        util: 'FO3Edit v4.0.4'
+  - name: 'RRFemalesRemoved.esp'
+    after: [ 'RaidersRedoneModular01.esp' ]
+    tag:
+      - Delev
```

You can use brackets when a list only contains a single item.
```suggestion
    tag: [ Delev ]
```